### ISSUE TITLE
Translating other table data

### ISSUE THREAD

**Issue by shuyafaye** [FabulousCupcake/umamusume-db-translate#17]
Is it possible if I put up translations for other table data? E.g _race_jikkyo_comment_

Since the current ones seems like it only replace _text_data_ only.

**Comment by FabulousCupcake** [comment]
Where does this text get used? Is it the race live commentary?  
Currently the tool only updates `text_data` table so it won't be as easy as adding a new csv file; need to update the page script.

**Comment by FabulousCupcake** [comment]
I'm concerned about the [space constraint in the live commentary](https://user-images.githubusercontent.com/25855364/113013477-07eee100-917c-11eb-83ac-b2c2be896ab6.png); it doesn't seem like it would show english text nicely. What do you think? 🤔

**Comment by shuyafaye** [comment]
Yeah, I tried and it does look horrible. Does text resizing works for all text display? I saw you used the _<size>_ tag before.

**Comment by FabulousCupcake** [comment]
Unity rich text _should_ work, but best way to find out is simply to try it out ingame.

**Comment by shuyafaye** [comment]
I figured out the characters voiceline's text is in _character_system_text_, but the text alignment and spacing is so damn hard to work with. High likely, this is because of the original japanese text file is fewer in characters compared to english. Any workaround to deal with this?

![test1](https://user-images.githubusercontent.com/26899667/113025756-e234fc80-91ba-11eb-81eb-ae2fa28da720.PNG)
![test2](https://user-images.githubusercontent.com/26899667/113025771-e6f9b080-91ba-11eb-879c-ffe51cd98b90.PNG)
![test3](https://user-images.githubusercontent.com/26899667/113025784-eb25ce00-91ba-11eb-8ff6-12f189458a80.PNG)

**Comment by GEEKiDoS** [comment]
> I figured out the characters voiceline's text is in _character_system_text_, but the text alignment and spacing is so damn hard to work with. High likely, this is because of the original japanese text file is fewer in characters compared to english. Any workaround to deal with this?
> 
> ![test1](https://user-images.githubusercontent.com/26899667/113025756-e234fc80-91ba-11eb-81eb-ae2fa28da720.PNG)
> ![test2](https://user-images.githubusercontent.com/26899667/113025771-e6f9b080-91ba-11eb-879c-ffe51cd98b90.PNG)
> ![test3](https://user-images.githubusercontent.com/26899667/113025784-eb25ce00-91ba-11eb-8ff6-12f189458a80.PNG)

Change text with hooking will not break the line:
![20210409231500](https://user-images.githubusercontent.com/14933408/114202124-7b1d0200-9989-11eb-8dae-2474dbe7e702.jpg)

I made a simple project to do that in these days:
https://github.com/GEEKiDoS/umamusume-localify

**Comment by shuyafaye** [comment]
@GEEKiDoS , can you provide more details on how to run yours for this patch?

**Comment by GEEKiDoS** [comment]
> @GEEKiDoS , can you provide more details on how to run yours for this patch?

Sure, just download version.dll from release or compile it your self(Requires VS2019, to generate project just click generate.bat)
and then grab config.json and localized_data folder from repo, then put all of them into umamusume game folder

**Comment by GoldElysium** [comment]
> > @GEEKiDoS , can you provide more details on how to run yours for this patch?
> 
> Sure, just download version.dll from release or compile it your self(Requires VS2019, to generate project just click generate.bat)
> and then grab config.json and localized_data folder from repo, then put all of them into umamusume game folder
> 
> edit: localized_data contains Chinese translation, use original_data then rename it to localized_data instead

Created a tool to convert this repo's translations to @GEEKiDoS's format. https://github.com/GoldElysium/umamusume-translation-converter
